ci/test_python.sh

```diff
@@ -7,7 +7,7 @@ set -euo pipefail
 
 . /opt/conda/etc/profile.d/conda.sh
 
-export RAPIDS_VERSION_NUMBER="24.02"
+export RAPIDS_VERSION_NUMBER="24.04"
```

Comment:
For more context, this environment variable was added relatively recently to fix a CI issue in 24.02 ( https://github.com/rapidsai/cucim/pull/680 ) after branching for 24.04. As a result the automerger picked up this version and pushed it along unchanged ( https://github.com/rapidsai/cucim/pull/681 ). Wonder if the automerger should run `ci/release/update-version.sh <VERSION>` as part of its updates to make sure values like this are updated

Reply:
Going to cherry-pick this commit so that we can have this for all PRs targeting `branch-24.04` ( https://github.com/rapidsai/cucim/pull/690 )